templates/zerver/integrations.html

```diff
@@ -101,128 +101,102 @@
             {% if integration.doc %}
                 <div id={{ integration.name }} class="integration-instructions">
                     {% include integration.doc %}
+                    <p style="font-size:11px; font-style:italic;">
+                        Logos are trademarks of their respective owners.
+                        None of the integrations on this page are created by,
+                        affiliated with, or supported by the companies
+                        represented by the logos.
+                    </p>
```

Comment:
I'd go with the version with splitting it to few files as the part of next commit here. What do you think?